packages/persist-state/src/lib/storage.ts

```diff
@@ -1,29 +1,36 @@
+import { isRecord } from '@ngneat/elf';
 import { Observable, of } from 'rxjs';
 
 export type Async<T> = Promise<T> | Observable<T>;
 
 export interface StateStorage {
-  getItem<T extends Record<string, any>>(
-    key: string
+  getItem<T extends Record<string, any> | string>(
+    key: string,
   ): Async<T | null | undefined>;
 
-  setItem(key: string, value: Record<string, any>): Async<any>;
+  setItem(key: string, value: Record<string, any> | string): Async<any>;
```

Comment:
Let's mark all these changes as breaking change in the commit description because I don't know how it'll affect people

Reply:
Added breaking changes info